src/webview_provider/docsEditPanel.ts

```diff
@@ -535,7 +543,7 @@ export class DocsEditViewPanel implements WebviewViewProvider {
           return undefined;
         }
 
-        const { command, syncRequestId, args } = message;
+        const { command, syncRequestId, ...params } = message;
```

Comment:
This might break other commands handled here. Make sure all the commands in this handler are tested from webview panel